packages/http-caching-proxy/src/http-caching-proxy.ts

```diff
@@ -225,18 +250,19 @@ export class HttpCachingProxy {
     // Since this proxy is for testing only, buffering the entire
     // response body is acceptable.
 
+    let data = backendResponse.data;
+    if (typeof data !== 'string') {
+      data = JSON.stringify(data);
```

Comment:
❗️ I find this problematic because we are reformatting JSON responses here. If the server responds with formatted JSON data (think of `JSON.stringify(value, null, 2)`), then our proxy will return JSON body with no formatting.

Even worse, if the original server returns non-JSON payload that's parsed by Axios (e.g. XML, see https://github.com/axios/axios/issues/667), then our proxy will change the response type! 

Please find a way how to forward the client request body and the backend response exactly as provided by the client and the backend, with no changes made by the proxy.

Reply:
Good catch. I forced axios to use `arraybuffer` now.